ports/stm32/boards/i2splltable.py

```diff
@@ -0,0 +1,220 @@
+"""
+This program computes I2S PLL parameters for STM32 
+processors supporting an I2S PLL in the clock tree.  
+Those processors are listed below in the mcu_support_i2spll_table[] list.
+"""
+
+from __future__ import print_function
```

Comment:
Unless you tested this with Python 2, I'd say we don't worry about Python 2 support and remove this line.

Reply:
I removed it.  Note that this line is present in `pllvalues.py`.